api/v1alpha4/conversion.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha4
+
+import (
+	conversion "k8s.io/apimachinery/pkg/conversion"
+	v1beta1 "sigs.k8s.io/cluster-api-provider-gcp/api/v1beta1"
+)
+
+// Convert_v1beta1_GCPMachineSpec_To_v1alpha4_GCPMachineSpec is an autogenerated conversion function.
+func Convert_v1beta1_GCPMachineSpec_To_v1alpha4_GCPMachineSpec(in *v1beta1.GCPMachineSpec, out *GCPMachineSpec, s conversion.Scope) error {
```

Comment:
Same as v1alpha3. Could we move this function to api/v1alpha4/gcpmachine_conversion.go?